app/assets/javascripts/components/profile/profile_cards.es6.jsx

```diff
@@ -1,5 +1,17 @@
 class ProfileCards extends Component {
 
+  static get propTypes() {
+    return {
+      currentUser: React.PropTypes.object.isRequired,
+    }
+  }
+
+  static get defaultProps() {
+    return {
+      currentUser: '',
```

Comment:
If this prop is an object (it should be!), then the default prop should be an empty object `{}`. Same for the other component!